### PR TITLE
React frontend

### PR DESCRIPTION
**Pull request by charanreddy24**
Two Changes in destake: 
1. app.py
2. requirements.txt
React-frontend files:

### CODE REVIEW

File: react-frontend/package-lock.json
Comment (sm86) [inline react-frontend/package-lock.json:1]: this file has to be on git ignore as well. We do not push this lock file

File: react-frontend/src/utils/constants.js
Comment (sm86) [inline react-frontend/src/utils/constants.js:1]: is this a temporary file, do we need to push it?

**Comment by sm86** [comment]
Can we also have a readme file for running in react frontend within this folder

File: react-frontend/src/utils/constants.js
Comment (charanreddy24) [reply react-frontend/src/utils/constants.js:1]: During the development phase, I faced difficulties fetching data from the backend. I added mock data for testing purposes. I will remove it.

**Review by sm86 — APPROVED** [review]
lgtm!

**Pull request merged**